source/shared/cpp/ObjectModel/MarkDownBlockParser.cpp

```diff
@@ -85,26 +85,26 @@ EmphasisParser::EmphasisState EmphasisParser::MatchText(EmphasisParser &parser,
         return EmphasisState::Captured;
     }
 
-    if (parser.IsMarkDownDelimiter(stream.peek()))
+    if (parser.IsMarkDownDelimiter(static_cast<char>(stream.peek())))
     {
         // encounterred first emphasis delimiter
         parser.CaptureCurrentCollectedStringAsRegularToken();
-        DelimiterType emphasisType = EmphasisParser::GetDelimiterTypeForCharAtCurrentPosition(stream.peek()); 
+        DelimiterType emphasisType = EmphasisParser::GetDelimiterTypeForCharAtCurrentPosition(static_cast<char>(stream.peek()));
         // get previous character and update the look behind if it was captured before 
         if (stream.tellg())
         {
             stream.unget();
-            parser.UpdateLookBehind(stream.get());
+            parser.UpdateLookBehind(static_cast<char>(stream.get()));
```

Comment:
Thanks for the fix. This one is not needed. UpdateLookBehind takes int.

Reply:
I'll double check this one once I get home. I only updated the code on the lines where   the os repo was asking for explicit casts, maybe I mistakenly updated this one.